server/src-lib/Hasura/Eventing/EventTrigger.hs

```diff
@@ -187,7 +187,7 @@ processEventQueue logger logenv httpMgr pool getSchemaCache eeCtx@EventEngineCtx
   where
     fetchBatchSize = 100
     popEventsBatch = do
-      let run = liftIO . runExceptT . Q.runTx pool (Q.RepeatableRead, Just Q.ReadWrite)
+      let run = liftIO . runExceptT . Q.runTx pool (Q.ReadCommitted, Just Q.ReadWrite)
```

Comment:
I think we should justify why `Q.ReadCommitted` is sufficient here, the description of the PR should suffice .